CONTRIBUTING.md

```diff
@@ -20,9 +26,14 @@ See Branching overview below - We use `dev` for our current development.
 
 In your fork from the root of the monorepo:
 
-- `npm install` - this will install Lerna at the root and allow the remaining scripts to be called.
-- (optional) `npm run install-git-hooks` - installs a pre-push hook that will lint all samples and apps before a `git-push`. Opt out per-push with the `--no-verify` flag.
-- `npm run reset` -> will clean all `node_modules`, re-install them with `lerna bootstrap`, and then build all JSS packages.
+- `yarn install` - this will install Lerna at the root, packages, samples and will link them and allow the remaining scripts to be called.
+- (optional) `yarn install-git-hooks` - installs a pre-push hook that will lint all samples and apps before a `git-push`. Opt out per-push with the `--no-verify` flag.
+- `yarn reset` -> will clean all `node_modules`, re-install them with `lerna bootstrap`, and then build all JSS packages.
+
+In our terms `workspace` - it's a sample or package.
```

Comment:
Does this mean _each_ sample and package is a workspace?

Reply:
Right